gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java

```diff
@@ -113,12 +120,18 @@ private void verifyBundle(String path) throws JSONException {
         var patientMigrationRequest = patientMigrationRequestDao.getMigrationRequest(patientNhsNumber);
         var expectedBundle = readResourceAsString(path).replace(NHS_NUMBER_PLACEHOLDER, patientNhsNumber);
 
-        assertBundleContent(patientMigrationRequest.getBundleResource(), expectedBundle);
+        FhirContext context = FhirContext.forDstu3();
+        IParser parser = context.newJsonParser();
+        var bundle = parser.parseResource(Bundle.class, patientMigrationRequest.getBundleResource());
+        assertBundleContent(patientMigrationRequest.getBundleResource(), expectedBundle,
+            Stream.of(generateJsonPathIgnores(bundle), STATIC_IGNORED_JSON_PATHS)
```

Comment:
Could you assign the result of the stream to a variable, and then pass the variable here? It would be more readable